R/utils.R

```diff
@@ -99,12 +99,16 @@ expose_stan_fns <- function(files, target_dir, ...) {
 #' functionality without needing to install `cmdstanr`.
 #'
 #' @param type A character string indicating the example to load.
-#' Supported options are "nowcast, "preprocessed_observations", "observations",
-#' and "script" which are the output of epinowcast()], [enw_preprocess_data()],
-#' and [enw_latest_data()] applied to the [germany_covid19_hosp] package
-#' dataset), and the script used to generate these examples respectively.
+#' Supported options are
+#'  * "nowcast, "preprocessed_observations", and "observations",
+#'  corresponding to the output of [epinowcast()], [enw_preprocess_data()],
+#' and [enw_latest_data()] applied to the [germany_covid19_hosp]
+#' dataset, and
+#'  * "scripts", the code used to generate these examples respectively.
 #'
 #' @return A `data.table` of summarised output
+#'  (`type` one of "nowcast, "preprocessed_observations", and "observations")
```

Comment:
This could probably do with just referring to the types of output available rather than listing as easy to miss this when adding new features.

Reply:
reasonable - have changed to read as `data.table` for everything but "scripts" argument.